### PR TITLE
colorset + test fixes

### PR DESCRIPTION
**Pull request by cota**
This adds the ability to specify the bars' colors directly in the .perf script.

While working on this, I fixed a few issues I ran into when trying to run the test suite.

Please have a look!

Thanks,

Emilio.

### CODE REVIEW

File: bargraph.pl [hunk 1]
```diff
@@ -942,6 +946,16 @@
 # the order here is the order for 9+ datasets
 $basefigcolor=32;
 $numfigclrs=0;
+
+# custom colorset
+if (@colorset) {
+    $colorset_base = $numfigclrs;
```
Comment (derekbruening) [inline bargraph.pl]: colorset_base does not seem to be used?

**Comment by derekbruening** [comment]
Thank you for the contribution!  I just see one issue, an unused variable.

**Comment by cota** [comment]
ouch, yes that's a remnant of something else I tried. If only we had enabled `use warnings` and `use strict` :)
I'll push a rebased branch without that line in a sec.

**Comment by cota** [comment]
*ping*

**Comment by derekbruening** [comment]
Sorry, forgot to check back.  Thanks again for the contribution.

**Pull request merged**